src/Common/ConfigTypes/ConfigTypes.jl

```diff
@@ -9,12 +9,16 @@ export ClimateMachineConfigType,
     AtmosLESConfigType,
     AtmosGCMConfigType,
     OceanBoxGCMConfigType,
+    OceanSplitExplicitConfigType,
     SingleStackConfigType
 
 abstract type ClimateMachineConfigType end
 struct AtmosLESConfigType <: ClimateMachineConfigType end
 struct AtmosGCMConfigType <: ClimateMachineConfigType end
 struct OceanBoxGCMConfigType <: ClimateMachineConfigType end
+struct OceanSplitExplicitConfigType <: ClimateMachineConfigType end
 struct SingleStackConfigType <: ClimateMachineConfigType end
 
+## WHY ISN'T THIS IN THE DRIVER FOLDER!?
```

Comment:
Yeah, this and `Utilities/Checkpoint` should really be in `Driver/`.

Reply:
Oh good, I was afraid there was going to be some arcane reason why this was better